lib/web_ui/lib/src/engine/text_editing/autofill_hint.dart

```diff
@@ -0,0 +1,112 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.6
+part of engine;
+
+/// Maps AutofillHints from the framework to the autofill hints that is used for
+/// browsers.
+/// See: https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/services/autofill.dart
+/// See: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete
+class BrowserAutofillHints {
+  static final BrowserAutofillHints _singletonInstance =
+      BrowserAutofillHints._();
+
+  /// The [BrowserAutofillHints] singleton.
+  static BrowserAutofillHints get instance => _singletonInstance;
+
+  Map<String, String> _flutterToEngineMap;
+
+  Map<String, String> _engineToFlutterMap;
```

Comment:
```suggestion
  final Map<String, String> _engineToFlutterMap = Map<String, String>();
```

Reply:
removed this one. I realized I didn't need to use a reverse map.